jprotoc/src/main/java/com/salesforce/jprotoc/ProtocPlugin.java

```diff
@@ -40,13 +41,25 @@ public static void generate(@Nonnull Generator generator) {
      * @param generators The list of generators to run.
      */
     public static void generate(@Nonnull List<Generator> generators) {
+        generate(generators, Collections.emptyList());
+    }
+
+    public static void generate(
+            @Nonnull List<Generator> generators, List<GeneratedExtension> extensions) {
         Preconditions.checkNotNull(generators, "generators");
         Preconditions.checkArgument(!generators.isEmpty(), "generators.isEmpty()");
+        Preconditions.checkNotNull(extensions, "extensions");
+
+        ExtensionRegistry extensionRegistry = ExtensionRegistry.newInstance();
```

Comment:
Please add a comment explaining what the `extensionRegistry` does.

Reply:
Done.